cmd/cluster/kubevirt/create.go

```diff
@@ -64,8 +66,14 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 		// As long as there is no official container image
 		// The image must be provided by user
 		// Otherwise it must fail
-		if opts.KubevirtPlatform.ContainerDiskImage == "" {
-			return errors.New("the container disk image for the Kubevirt machine must be provided by user (\"--containerdisk\" flag)")
+		if opts.KubevirtPlatform.ContainerDiskImage != "" && opts.KubevirtPlatform.ContainerDiskRepo != "" {
+			return errors.New("only one of \"--containerdisk\" and \"--containerdisk-repository\" can be specified")
+		}
+		if opts.KubevirtPlatform.ContainerDiskImage == "" && opts.KubevirtPlatform.ContainerDiskRepo == "" {
+			return errors.New("the container disk image for the Kubevirt machine must be provided by user (\"--containerdisk\" or \"--containerdisk-repository\" flag)")
+		}
+		if opts.KubevirtPlatform.ContainerDiskRepo != "" {
+			opts.KubevirtPlatform.ContainerDiskImage = fmt.Sprintf("%v:%v", opts.KubevirtPlatform.ContainerDiskRepo, util.RHCOSOpenStackChecksumParameter)
```

Comment:
if neither the container or the repo are provided, can we just default to using the datavolume http import?